documentation/tutorials/text-search-semantic.md

```diff
@@ -0,0 +1,317 @@
+---
+# Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+title: "Semantic Text Search: Quick start "
+---
+
+The main goal of this tutorial is to investigate and report on the possibility of providing out of the box semantic search capabilities to text applications with as little application specific tunning as possible. In order to do that, we will first describe the data used, the metrics of interest and important baselines that will be used to judge our results. 
```

Comment:
Not sure "... and report ..." is what a tutorial is for. Probably just drop that.